test/Microsoft.AspNet.Mvc.Xml.Test/XmlAssertTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+using Xunit.Sdk;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    public class XmlAssertTest
+    {
+        [Theory]
+        [InlineData("<A>hello</A>", "<A>hey</A>")]
+        [InlineData("<A><B>hello world</B></A>", "<A><B>hello world!!</B></A>")]
+        [InlineData("<A>hello<B>world</B>hello</A>", "<A>hello<B>world!</B>hello</A>")]
+        public void ValidatesTextNodes(string input1, string input2)
+        {
+            Assert.Throws<EqualException>(() => XmlAssert.Equal(input1, input2));
```

Comment:
check `Message` here matches what you expect.  I expect it will mention `<A>hellohello<B>world</B></A>` and ``<A>hellohello<B>world!</B></A>`